src/components/ui/button.tsx

```diff
@@ -5,7 +5,7 @@ import { cva, type VariantProps } from "class-variance-authority"
 import { cn } from '~/utils/cn'
 
 const buttonVariants = cva(
-  "inline-flex items-center justify-center whitespace-nowrap rounded-md text-sm font-medium ring-offset-background transition-colors focus-visible:outline-none focus-visible:ring-2 focus-visible:ring-ring focus-visible:ring-offset-2 disabled:pointer-events-none disabled:opacity-50",
+  "inline-flex items-center justify-center whitespace-nowrap rounded-md text-sm font-medium ring-offset-background transition-colors focus-visible:outline-none focus-visible:ring-2 focus-visible:ring-ring focus-visible:ring-offset-2 disabled:pointer-events-none disabled:opacity-50 transition duration-500",
```

Comment:
I just want to make sure this was an intentional global change you want to make to the Button component. 

As a general rule the primitive components in the components directory are "global", meaning that once you change something in any of these files its going to cascade down to ALL other components. 


If you need like a specific change to button for a single instance, you can always just append more classnames to your usecase 

ie. `<Button className=''[overriding styles here]" >Thingy</Button> `

Reply:
@iamtheozzy This was something I was trying out at the time, and I was also unaware you could override the Button styling with a className. I assumed you had to do all of your styling inside the component file itself. I am going to remove this since I really dont think this was something I wanted to push, and was more so just a `play around and see`.

Reply:
@vcristian1 okay for sure! yea its good to figure those things out. Think of the primitive components as the most generic version of components. You can override almost anything about them using tailwind and classes. Which is why its such a powerful pattern.